vllm_ascend/models/deepseek_v2.py

```diff
@@ -465,20 +465,21 @@ def forward(
             hidden_states: torch.Tensor,
             kv_cache: Optional[torch.Tensor] = None,
             attn_metadata: Optional[AttentionMetadata] = None) -> torch.Tensor:
+        enable_multistream_mla = (self.enable_multistream_mla
```

Comment:
nit: we'd better log here to notify users why enable_multistream_mla not enable here and add some note about this.